app/services/ccms/attribute_value_generator.rb

```diff
@@ -311,10 +303,10 @@ def benefit_check_passed?(_options)
     end
 
     def proceeding_cost_limitation(options)
-      application_proceeding_type = options[:appl_proceeding_type]
-      return 'MULTIPLE' if application_proceeding_type.used_delegated_functions?
+      proceeding = options[:proceeding]
+      return 'MULTIPLE' if proceeding.used_delegated_functions_on.present?
```

Comment:
Can you use the `used_delegated_functions?` method on the proceeding here instead of calling `present?`?
```suggestion
      return 'MULTIPLE' if proceeding.used_delegated_functions?
```